commands/init.go

```diff
@@ -45,6 +46,13 @@ var InitCmd = &cli.Command{
 			EnvVars:     []string{"LILY_SNAPSHOT"},
 			Destination: &initFlags.importSnapshot,
 		},
+		&cli.IntFlag{
+			Name:        "backfill-tipsetkey-range",
+			Usage:       "Determine the extent of backfilling from the head.",
+			EnvVars:     []string{"LILY_BACKFILL_TIPSETKEY_RANGE"},
+			Value:       1800,
```

Comment:
our snapshot range is 2890 epochs. Should we make the default something like 3000 or 3600 to ensure we always backfill more than a day?

Reply:
changed